suite-common/wallet-core/src/fiat-rates/fiatRatesReducer.ts

```diff
@@ -54,7 +53,7 @@ export const prepareFiatRatesReducer = createReducerWithExtraDeps(
                     ...currentRate,
                     rate: action.payload.rate,
                     lastTickerTimestamp: (action.payload.lastTickerTimestamp * 1000) as Timestamp,
-                    lastSuccessfulFetchTimestamp,
+                    lastSuccessfulFetchTimestamp: Date.now() as Timestamp,
```

Comment:
This should be passed from action creator

Reply:
fe3542754bf3bb5c258deb4e7a7fcf2130a252fc